bin/oref0-bluetoothup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+if ( ifconfig wlan0 | grep -q "inet addr" ) && ( ifconfig bnep0 | grep -q "inet addr" ); then
+   sudo killall bluetoothd
```

Comment:
Do we really want to kill Bluetooth just because we got a wifi IP? Shouldn't we test Internet connectivity over it, and only disconnect the bt-pan (not kill Bluetooth) if wifi has access to the Internet?

Maybe this part should be a new PR, as it's not pancreabble related and might require some more discussion about desired behavior.